componets/buttons/buttons.css

```diff
@@ -0,0 +1,106 @@
+.btn {
+  margin: 8px;
+  padding: 15px;
```

Comment:
If possible please use relative units.

Reply:
padding and margin is less than 1 rem so I thought  it is better to keep in fixed and I don't  want margin to increase  with rem units , is it fine